packages/sync/README.md

```diff
@@ -123,12 +130,58 @@ exampleMutation(...) @noSquash {
 }
 ```
 
-## Designing your types
+## Conflicts
 
-When designing your GraphQL schema types `id` field is always required.
-Library will perform business logic assuming that `id` field will be supplied and returned from server. Without this field some offline functionalities will not work properly.
+TODO
 
-# Contribution guide
+### Conflict strategies
```

Comment:
I know you have a TODO, but we should probably mention versions somewhere here.

Reply:
Yeah you are right, it does make sense to wait. The only reason I pointed it out was the part discussing conflict resolution strategies doesn't have much context without it. Happy to wait until we know exactly what it will look like though.